eWonConnector-build/pom.xml

```diff
@@ -25,7 +25,7 @@
             <plugin>
                 <groupId>com.inductiveautomation.ignitionsdk</groupId>
                 <artifactId>ignition-maven-plugin</artifactId>
-                <version>1.1.0</version>
+                <version>1.2.0</version>
```

Comment:
Version is changing in this commit but the commit message doesnt indicate that is in scope for this commit

Reply:
Updated msg